apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -3788,13 +3774,19 @@ open_websockets_count() ->
                  QName =:= QueueName]).
 
 sign_and_post_tx(EncodedUnsignedTx) ->
+    sign_and_post_tx(EncodedUnsignedTx, [{mempool_check, true}]).
+
+sign_and_post_tx(EncodedUnsignedTx, Opts) ->
     {ok, SerializedUnsignedTx} = aec_base58c:safe_decode(transaction, EncodedUnsignedTx),
     UnsignedTx = aetx:deserialize_from_binary(SerializedUnsignedTx),
     {ok, SignedTx} = rpc(aec_keys, sign, [UnsignedTx]),
     SerializedTx = aetx_sign:serialize_to_binary(SignedTx),
     {ok, 200, _} = post_tx(aec_base58c:encode(transaction, SerializedTx)),
     % create_contract_tx is in mempool
-    {ok, [SignedTx]} = rpc(aec_tx_pool, peek, [infinity]), % same tx
+    case proplists:get_value(mempool_check, Opts) of
+        true -> {ok, [SignedTx]} = rpc(aec_tx_pool, peek, [infinity]); % same tx
```

Comment:
This may be replaced with call(s) to user HTTP API. I am thinking of [get_account_transactions with pending txs](https://github.com/aeternity/epoch/blob/e2589afebe6acefaddb425ca77768bdc87db9a33/config/swagger.yaml#L1179) checking [height `-1`](https://github.com/aeternity/epoch/blob/e2589afebe6acefaddb425ca77768bdc87db9a33/apps/aetx/src/aetx_sign.erl#L158).

Reply:
Replaced in 45c0d3d.